samples/snippets/csharp/VS_Snippets_WebNet/System.Configuration.ConfigurationBuilder/CS/SampleConfigurationBuilder.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Configuration;
+using System.Xml;
+
+namespace Samples.AspNet.Config
+{
+
+   public class SampleConfigurationBuilder : ConfigurationBuilder
+    {
+
+        public override XmlNode ProcessRawXml(XmlNode rawXml) 
+        {
+
+            string rawXmlString = rawXml.OuterXml;
+
+            if (String.IsNullOrEmpty(rawXmlString)) {
+             
+                return rawXml;
+
+            }
+
+            rawXmlString = Environment.ExpandEnvironmentVariables(rawXmlString);
+
+            XmlDocument doc = new XmlDocument();
+            doc.PreserveWhitespace = true;
+            doc.LoadXml(rawXmlString);
+            return doc.DocumentElement;
+
+        }
+
+        public override ConfigurationSection ProcessSectionInstance(ConfigurationSection configSection) 
```

Comment:
This would be a great use case for expression bodied members:

```csharp
public override ConfigurationSection ProcessSectionInstance(ConfigurationSection configSection) => configSection;
```